CONTRIBUTING.md

```diff
@@ -32,6 +32,10 @@ make nix-fish # Runs fish shell in the nix environment
 
 ### OPAM
 
+**Prerequisites**:
+- Install the [opam](https://opam.ocaml.org/) package manager
+- Install `tree` command line tool (`brew install tree` for macOS or `apt install tree` for Linux)
```

Comment:
Maybe add optional item for manually installing odoc?

Reply:
Added in 6ac50ed.